### ISSUE TITLE
What if make eslint plugin based on vue-type-check?

### ISSUE THREAD

**Issue by nibytes** [Yuyz0112/vue-type-check#2]
Subj

**Comment by kenberkeley** [comment]
Yes, hope it can be integrated with Vue CLI generated projects easily and smoothly.

**Comment by Yuyz0112** [comment]
Since the implementation is based on vetur which runs relatively slow without incremental checking support, I'm not sure it could be integrated with real-time lint tools easily.

Currently, I'm also using it in the commit hooks.
